gemfiles/rails_edge.gemfile

```diff
@@ -0,0 +1,34 @@
+# This file was generated by Appraisal
+
+source 'https://rubygems.org'
+
+gem 'rails', '~> 5.0.0'
```

Comment:
Should we add this `appraise` into `Appraisal`? And I guess we should use edge of github.
it should be something like `gem 'rails', github: 'rails/rails'`